src/generator/lib/util.py

```diff
@@ -1199,7 +1199,7 @@ def unique(
     original: str,
     desired: str,
     attempts: int = 0,
-    assigned: dict[str, str] = {},
+    assigned: Dict[str, str] = {},
```

Comment:
Could you educate me on this change?

Reply:
Oh, as the README suggested, I used python 3.8.19 but it was not compiling because of the wrong type.
The newer versions use `dict` while the older versions use `Dict`, same goes with `List`. 

In order to do `make`, I had to make this change so it works are expected with python 3.8.

I might be wrong here but I added this for compatibility with the README and the python version. We can remove this but I don't know if other developers will face the same issue.

Can you confirm the python version on your machine and if the `make` works?

Reply:
I assume that older python interpreters are also fine with `Dict`, as otherwise this would be a breaking change in Python itself.
So let's keep what's most compatible.

Reply:
Could you respond to this?
Will `Dict` be backward compatible?

Reply:
Sorry, I thought you wanted to keep this change. Yes, `Dict` is backwards compatible but `dict` is not. So, people using python 3.9+ should not have any issues as I have checked with version 3.8 and 3.12.

Reply:
Great, thank you!